src/Core/System/StateMachine/Aggregation/StateMachineHistory/StateMachineHistoryEntity.php

```diff
@@ -32,7 +32,7 @@ class StateMachineHistoryEntity extends Entity
 
     protected ?StateMachineStateEntity $toStateMachineState = null;
 
-    protected string $userId;
+    protected ?string $userId;
```

Comment:
Would do something like this. It's more error prove, I guess
```suggestion
    protected ?string $userId = null;
```

Otherwise, this kind of code will fail.
```php
(new StateMachineHistoryEntity())->getUserId();
```

Reply:
Yep, sounds good.